bundles/org.openhab.binding.warmup/pom.xml

```
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.8-SNAPSHOT</version>
```

Can you rebase to 2.5.9. With `git pull --rebase` and than `git push --force-with-lease` to get the changes back to GitHub
```suggestion
    <version>2.5.9-SNAPSHOT</version>
```

---

Migrated to v3.0.0